### DISCUSSION TITLE
New Getting started video with pip install?

### DISCUSSION THREAD

**Discussion by BradMBD** [jgerstmayr/EXUDYN#37] [Q&A on usage of Exudyn]
Dear Prof. Gerstmayr, I found EXUDYN now cannot be installed in the way you showed in the getting started youtube video (Aug. 2020). I installed the Anaconda3-5.2.0-Windows-x86_64 (Python 3.6.5) as mentioned in theDoc.pdf. Then, 

I first tried PIP INSTALLER. It turns out it doesn't work. I tried to import exudyn, the spyder console shows "ModuleNotFoundError: No module named 'exudyn'".

Then, I downloaded the "exudyn-1.6.0-cp36-cp36m-win_amd64.whl" that matches the python version. I installed via wheel files according to the tutorial in theDoc.pdf. I tried to import exudyn again in Spyder, it shows 

"ImportError: Import of exudyn C++ module failed; check 32/64 bits versions, restart your iPython console or try to uninstall and install exudyn". But, I am sure my labtop is 64bit and the version of Anaconda/python and exudyn are consistent (the recommended one in theDoc.pdf). So, could you please help give any suggestions?

Or, could you please make a updated getting started video tutorial?

Thanks,

Tengfei

**Comment by jgerstmayr** [comment]
already two months ago, there are new youtube videos with the updates workflow! 
[https://www.youtube.com/watch?v=i9_yxSLndfA&list=PLZduTa9mdcmOh5KVUqatD9GzVg_jtl6fx&index=32&t=25s](url)
